files/k8s-resource/calico-all.yaml

```diff
@@ -218,13 +265,25 @@ spec:
               value: "autodetect"
             # Enable IPIP
             - name: CALICO_IPV4POOL_IPIP
-              value: "Always"
+              value: "Never"
```

Comment:
Discussed here https://gigantic.slack.com/archives/C01BYMF6RN0/p1629140646152200 and here https://gigantic.slack.com/archives/C3S482Z98/p1629144487000400

Reply:
Seems reasonable to me - only question:
Is there something we can set as a target to verify it works as intended? E.g. we upgrade some dev clusters and monitor how they are behaving more closely?

Reply:
Like prometheus monitoring? I think `net-exporter` should already alert us if some pods can't reach other pods.

Reply:
Yes - just keeping a closer eye on the before & after.